### ISSUE TITLE
Dragging works only once on chrome

### ISSUE THREAD

**Issue by IamFonky** [react-dnd/react-dnd#1476]
**Describe the bug**
I'm making a toolbar with draggrable tabs. When I drag the tab the first time it's working but the second time the drag dosen't work anymore.

**Reproduction**

[Live Reproduction](https://codesandbox.io/s/react-dnd-fail-2pxpu)

1. Drag a tab
2. Drop it on a drop target
3. Try dragging a tab again
4. No console error but impossible to drag again any of the tabs

**Expected behavior**
It should work always, or at least throw an error when fails.

**Screenshots**
[Live Reproduction](https://codesandbox.io/s/react-dnd-fail-2pxpu)

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Linux Mint 18.3, Linux Mint 19.1, Ubuntu 18.04.2
- Browser [e.g. chrome, safari] Chrome (only)
- Version [e.g. 22] Version 73.0.3683.75 (Official Build) Built on Ubuntu

**Comment by IamFonky** [comment]
I've updated the example with a working and failing mode.
It seem to fail when the drop element doesn't have children.

**Comment by IamFonky** [comment]
I think i found a fail pattern.
It seem to fail when the drag source is moved away from the cursor at the beginning of the drag.
![dndbug](https://user-images.githubusercontent.com/25899984/61630097-1d152c00-ac87-11e9-8c2e-8d6f4e8f2aff.png)
I added a workaround which consist in waiting 100 ms before changing style while dragging... ugly but working great

**Comment by IamFonky** [comment]
Oh come on!
There's only 50 issues and I did my best to show the exact problem.
If I can help just tell me...

**Comment by Dodie324** [comment]
Not sure why this was marked as stale. 🤷‍♂ 

Any way, I'm in a similar boat as @IamFonky . I have a rich UI that consists of draggable nodes. Some of the nodes can be dragged around the UI just fine; however, there are a handful that don't. I've confirmed that this behavior only exists in Chrome and Safari, which leads me to believe it's a Webkit issue.

I tried doing as Dan suggested [here](https://github.com/react-dnd/react-dnd-html5-backend/issues/18#issuecomment-183770505) a while back, but had no luck. I'm not sure what to do at this point.

![Screen Recording 2019-10-10 at 09 21 AM](https://user-images.githubusercontent.com/9405172/66572758-86e2fa80-eb3f-11e9-826a-d92277502aa4.gif)

**Comment by nearwood** [comment]
Can confirm this on
```
Chrome - Version 77.0.3865.120 (Official Build) (64-bit)
Chome Canary - Version 80.0.3948.0 (Official Build) canary (64-bit)
Edge Dev - Version 79.0.308.1 (Official build) dev (64-bit)
```

Edge seems to work slightly different, and more often than not. With the code sandbox example linked above: Chrome sometimes works. With me, when it doesn't work, sometimes it fails to start the drag at all, and sometimes it triggers it briefly (drop zone flashes on then off real quick).

Which is similar to the behavior I'm seeing in our app.

**Comment by nearwood** [comment]
Can confirm this on
```
Chrome - Version 77.0.3865.120 (Official Build) (64-bit)
Chome Canary - Version 80.0.3948.0 (Official Build) canary (64-bit)
Edge Dev - Version 79.0.308.1 (Official build) dev (64-bit)
```

Using the code sandbox example linked above: Edge seems to work slightly different, and more often than not. Chrome sometimes works. When it doesn't work, sometimes it fails to start the drag at all, and sometimes it triggers it briefly (drop zone flashes on then off real quick).

The latter issues with Chrome are similar to the behavior I'm seeing in our app.

**Comment by raq929** [comment]
I am also having this issue on Chrome.

**Comment by Dietermartens** [comment]
Same issue here!

**Comment by leefsmp** [comment]
Same issue, super annoying as it took me a day to implement a dnd feature only to realize this issue makes the whole thing unusable...

**Comment by lpolito** [comment]
Had the same issue as described. Turns out the explanation @IamFonky provided (https://github.com/react-dnd/react-dnd/issues/1476#issuecomment-513763872) was the same issue I was having. Consequently his solution worked perfectly.

> I added a workaround in the demo which consist in waiting 100 ms before changing style while dragging... ugly but working great

**Comment by thomas-lab** [comment]
Yup. Somehow updating the rendering based on `canDrop`/`isDragging` can make it fail silently.
As you said, adding a timer works. Another solution is just to wait for the next tick.

A way to perform that in React is:
``` typescript
const [isDraggingDelayed, setIsDraggingDelayed] = React.useState(false);
React.useEffect(() => {
    setImmediate(() => setIsDraggingDelayed(isDragging));
}, [isDragging]);
```
And then using `isDraggingDelayed` in your rendering function will work fine.

**Comment by lamvananh** [comment]
i have developed a page similar to this example:
 https://codesandbox.io/s/github/react-dnd/react-dnd/tree/gh-pages/examples_hooks_js/00-chessboard?from-embed=&file=/src/Knight.jsx:257-263
But in my app, it's only can drag once on google chorme (on Firefox it's working well) until i removed this line
`{!isOver && canDrop && <Overlay class='overlay' color="yellow" />} `
in file Knight.jsx. It's can drag multi time on google chorme now.
But I can't not understand why this happend.

**Comment by sielay** [comment]
> codesandbox.io/s/epic-gates-fy95w

this worked perfectly, thanks @thomas-lab
